### DISCUSSION TITLE
Set Default live transcode or change priority

### DISCUSSION THREAD

**Discussion by slimthick** [stashapp/stash#4173] [Q&A]
My WMV files seek and load faster when using DASH/HLS instead of the default mp4 live transcode. Is there a way to set the default transcode or reorder the list in the player? I checked the yaml config file as well and couldn't find anything.

**Comment by dingdongsolong4** [comment]
This is not currently possible, but would be a useful feature - feel free to make a feature request.

**Comment by feederbox826** [comment]
Was converted into issue https://github.com/stashapp/stash/issues/4175
